### ISSUE TITLE
Fix the depth of the ontology page list

### ISSUE THREAD

**Issue by jmillanacosta** [NanoCommons/erm-database#20]
The ontology page hierarchy now has a max depth of 1. 
![image](https://github.com/NanoCommons/erm-database/assets/83466805/2cd14d12-74a3-4819-bda3-dec847b51886)

It has to be fixed into a Protege-like indentation for class hierarchy.

**Comment by egonw** [comment]
Yeah, I think of doing it like in this issue: https://github.com/NanoCommons/erm-database/issues/19

**Comment by jmillanacosta** [comment]
d3 https://github.com/ammar257ammar/PRA3006-D3-Examples/blob/main/TreeChart/index.html
